src/components/molecules/LogInOut.js

```diff
@@ -1,15 +1,21 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { ActionButton } from "../atoms/ActionButton";
+import { Alink } from "../atoms/Alink";
 
 /**
  * This component show login or a logout button according to the user authenticated flag
  */
 export function LogInOut(props) {
+  //Link for user
+  let url = "https://169.59.166.63:9044/ua/";
```

Comment:
never hardcode variables like this...use variable from [variable.js](https://github.com/DTS-STN/ServicePrototypeFrontend/blob/main/src/variables.js). Ideally this should be a prop and should be passed in from the Page component

Reply:
I changed it now to pass it through props.

Reply:
you may want to use the props directly instead of thru a variable. it is a suggestion only